picasso/src/main/java/com/squareup/picasso3/Picasso.java

```diff
@@ -156,7 +159,9 @@ void onImageLoadFailed(@NonNull Picasso picasso, @NonNull Uri uri,
     this.closeableCache = closeableCache;
     this.cache = cache;
     this.listener = listener;
+    this.imageDecoderFactory = imageDecoderFactory;
```

Comment:
when you call newBuilder().addImageDecoder(...).build(), are the old image decoders retained and appended to? (add a test?)

Reply:
Fixed in 7efb773